src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -82,7 +82,8 @@ public enum ErrorCode {
   SHRED_DOC_LIMIT_VIOLATION("Document size limitation violated"),
 
   EXISTING_COLLECTION_DIFFERENT_SETTINGS("Collection already exists"),
-
+  EXISTING_TABLE_NOT_DATA_API_COLLECTION(
+      "An existing table does not follow Data API collection schema pattern"),
```

Comment:
nit: An existing -> Existing 
(I am not native speaker and articles are confusing, but here it's a specific table so indefinite "an" seems misplaced)

Reply:
Maybe change to: "Existing table not valid Data API Collection"

Reply:
Got it, solved